src/components/reportPage/BatchJob.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useCallback, useEffect } from "react";
+import StepUpload from "./StepUpload";
+import StepEmail from "./StepEmail";
+import "./batch-job.css";
+
+const BatchJob = () => {
+  const FILE_LIMIT = 100;
+  const [currentStep, setCurrentStep] = useState(1);
+  const [files, setFiles] = useState([]);
+  const [email, setEmail] = useState();
+  const [emailVerified, setEmailVerified] = useState(false);
+  const [formData, setFormData] = useState({
+    email: "",
+    emailVerified: false,
+  });
+  const handleChange = (event) => {
+    // setFormData({
+    //   ...formData,
+    //   [event.target.name]: event.target.value,
+    // });
+  };
```

Comment:
Is this function used anywhere?

Reply:
maybe in the future might use so kept it